services/member-management-client/src/index.js

```diff
@@ -1,7 +1,19 @@
 import ReactDOM from 'react-dom';
 import React, { Fragment } from 'react';
+import * as MembersService from './members-service';
 
 class MemberManagementPage extends React.Component {
+  constructor(props) {
+    super(props)
```

Comment:
TIL: I should be calling super in constructors

> The constructor for a React component is called before it is mounted. When implementing the constructor for a React.Component subclass, you should call super(props) before any other statement. Otherwise, this.props will be undefined in the constructor, which can lead to bugs.